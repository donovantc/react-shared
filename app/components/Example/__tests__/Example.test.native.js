import React from 'react'
import Example from '../'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Example />).toJSON()

  expect(tree).toMatchSnapshot()
})
