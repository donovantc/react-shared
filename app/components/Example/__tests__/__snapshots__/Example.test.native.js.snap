// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontWeight": "bold",
        "paddingVertical": 5,
        "textAlign": "center",
      }
    }
  >
    What is your name?
  </Text>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    style={
      Object {
        "borderColor": "#114183",
        "borderWidth": 2,
        "height": 50,
        "marginVertical": 20,
      }
    }
    value=""
  />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#38c742",
        "borderRadius": 5,
        "height": 50,
        "justifyContent": "center",
        "opacity": 1,
        "padding": 10,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontWeight": "bold",
          "paddingVertical": 5,
          "textAlign": "center",
        }
      }
    >
      Submit
    </Text>
  </View>
</View>
`;
